earthaccess/store.py

```diff
@@ -153,7 +163,7 @@ def make_instance(
     ):
         # NOTE: This uses the first data_link listed in the granule. That's not
```

Comment:
I wonder if this note needs to be revised, if I recall correctly there was a mapping granule to URLs method to zip them and all of the assets/files in the granue under GET_DATA are opened. Testing HSL or some of the collections that have multiple files will clarify the behavior.